test/e2e/saucelabs.conf.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-console, no-process-env */
+
+const config = require('config');
+const fileAcceptor = require('test/file_acceptor');
+const supportedBrowsers = require('./crossbrowser/supportedBrowsers.js');
+
+const evidenceUploadEnabled = config.get('features.evidenceUpload.enabled');
+const tunnelName = process.env.SAUCE_TUNNEL_IDENTIFIER || config.get('saucelabs.tunnelId');
+
+const getBrowserConfig = browserGroup => {
+  const browserConfig = [];
+  for (const candidateBrowser in supportedBrowsers[browserGroup]) {
+    if (candidateBrowser) {
+      const desiredCapability = supportedBrowsers[browserGroup][candidateBrowser];
+      desiredCapability.tunnelIdentifier = tunnelName;
+      desiredCapability.tags = ['sscs'];
+      browserConfig.push({
+        browser: desiredCapability.browserName,
+        desiredCapabilities: desiredCapability
+      });
+    } else {
+      console.error('ERROR: supportedBrowsers.js is empty or incorrectly defined');
+    }
+  }
+  return browserConfig;
+};
+
+const setupConfig = {
+  tests: './smoke/appeal.test.js',
```

Comment:
I guess we run this test just for now to check it all works fine and once that confirms we may want to add a more complete batch. Is that correct?

Reply:
Thats a good point, I had been thinking of restricting the crossbrowser tests to just a single test that covers all / most of the pages, but I'm not familiar enough with SSCS yet to say which test that might be

Reply:
If you have any ideas of a more suitable test / couple of tests then more than happy to change to use those instead

Reply:
let's leave it as it is just for now, yes. :)